doc/user_manual/chSpecialElements.tex

```diff
@@ -1543,40 +1547,39 @@ \section{Scattering} \label{sec:scatter}
 \bigskip
 \noindent\texttt{PRO name GAUSS1 beamtot[particles] sigmaX[mm] sigmaY[mm] offsetX[mm] offsetY[mm]} \\
 
-The \texttt{GAUSS1} profile type us given by
+The \texttt{GAUSS1} profile type is given by
 \begin{align}
     \rho(x,y) = \frac{N_{\mathrm{tot}}}{2\pi\sigma_x\sigma_y}
                 \exp\left(-\frac{(x-\mu_x)^2}{2\sigma_x^2}\right)
                 \exp\left(-\frac{(y-\mu_y)^2}{2\sigma_y^2}\right).
 \end{align}
 
-\paragraph{GENerator} \texttt{GEN name type (arguments)} \\
+\paragraph{Event Generator}~ \texttt{GEN name type [arguments]} \\
 
 The generator block takes a name and a generator type input, followed by the parameters for the
 generator type.
 
 \bigskip
-\noindent\texttt{GEN name PPBEAMELASTIC a b1 b2 phi tmin (crossSection)} \\
+\noindent\texttt{GEN name PPBEAMELASTIC a b1 b2 phi tmin [crossSection]} \\
 
 Takes five or six input arguments, and generates the probability distribution given by
 \begin{align}
     g(t) = \frac{1}{a_1^2}\frac{d\sigma}{dt} = e^{-b_1 t}+ 2ae^{-(b_1+b_2)t/2}\cos{\phi} + a^2e^{-b_2 t},
 \end{align}
 where the first expression is a soft scatter data fit, the third expression a hard scatter fit, and the second expression is the interference. $a = a_2/a_1$ is the amplitudes of the expressions.\index{elastic scattering}
 These are combined into the first four input arguments $a$, $b_1$, $b_2$, and $\phi$, as well as $t_{min}$ which provides a cut-off limit.
-The optional sixth argument defines a fixed cross section for the scattering probability.
+The optional sixth argument defines a fixed cross section for the scattering probability in units of millibarn.
```

Comment:
How is the cross-section computed when this is not given?

Reply:
Cross section  is no longer optional. Need to correct this.